<launch>
    <arg name="service_name" default="/trigger_service"/>
    <node pkg="task03" exec="trigger_node" name="trigger_node" output="screen">
        <param from="$(find-pkg-share task03)/config/task03.yaml"/>
        <param name="service_name" value="$(var service_name)"/>
    </node>
</launch>
